ee/codegen/src/__test__/nodes/__snapshots__/inline-prompt-node.test.ts.snap

```diff
@@ -673,7 +685,7 @@ class PromptNode(InlinePromptNode):
         JinjaPromptBlock(
             state="ENABLED",
             cache_config=None,
-            template="Summarize what this means {{ INPUT_VARIABLE }}",
+            template="""Summarize what this means {{ INPUT_VARIABLE }}""",
```

Comment:
If there are no `\n` characters in the string, we keep it on one line.

Reply:
pretty neat